github/orgs.go

```diff
@@ -70,6 +70,11 @@ type Organization struct {
 	ReposURL         *string `json:"repos_url,omitempty"`
 }
 
+type OrganizationInstallations struct {
```

Comment:
Please comment this struct for the auto-generated Godocs.

Reply:
Done